packages/opentelemetry-rca-metrics/README.md

```diff
@@ -0,0 +1,54 @@
+#OpenTelemetry RCA Metrics for Node.js
```

Comment:
The whole package including readme assumes the user already knows what RCA means. I think more explanation is needed as to what is collected.